src/openms/include/OpenMS/ANALYSIS/ID/SiriusAdapterAlgorithm.h

```diff
@@ -49,6 +49,13 @@ namespace OpenMS
       /// default constructor
       SiriusAdapterAlgorithm();
 
+      struct OPENMS_DLLAPI SiriusWorkspaceIndex
+      {
+      public:
+        int array_index;
+        int scan_index;
+      };
+
       /// Struct for temporary folder structure
       struct OPENMS_DLLAPI SiriusTemporaryFileSystemObjects
```

Comment:
we would use class instead of struct

Reply:
That is just preference and has no deeper meaning, or?